pushy/src/test/java/com/eatthepath/pushy/apns/util/ApnsPayloadBuilderTest.java

```diff
@@ -508,7 +508,7 @@ void testSetRelevanceScore() {
     }
 
     @ParameterizedTest
-    @ValueSource(doubles = {-7, 1.1, Double.NaN, Double.NEGATIVE_INFINITY, Double.POSITIVE_INFINITY})
+    @ValueSource(doubles = {Double.NaN})
```

Comment:
Very minor: I think we can make this a normal `@Test` instead of a `@ParameterizedTest` if there's only one value we're concerned about.